### DISCUSSION TITLE
Force Refresh of Alexa Device Names?

### DISCUSSION THREAD

**Discussion by vaderag** [alandtse/alexa_media_player#1983] [General]
We've moved a few devices around the house, and (for example) my TV_Echo_Dot has become Study, Study has become Kids_Room etc but the integration doesn't seem to have picked up the changes and its been a couple of days and a couple of restarts

I'm assuming these are simply set at creation and maybe I just need to rename them all manually - e.g. media_player.tv_echo_dot is still the name of the Study? 

That said, I'm wondering if there is somethings more, since I also have a 3 NEW devices that've been added but only the "do not disturb switch" - linked to an unknown device - how can I get it to pull the full device? 

Probably missing something obvious but I've scanned the wiki and can't see anything that I might need to do that I've missed!

Thank you

**Comment by pot-noble** [comment]
I'm not faced this yet but definitely will in future. So, I'm also would love to hear an answer for that if someone have one.
Maybe re-installing the integration and re-discovering devices will do the trick but that would mean re-configuring all automations, scripts etc that using any entities from AMP which obviously would take time if you have lots of automatons.
